src/NuGet.Clients/NuGet.Tools/NuGet.Tools.csproj

```diff
@@ -334,13 +344,44 @@
     </None>
   </ItemGroup>
   <ItemGroup />
-  <Target Name="AfterBuild">
+  <Target Name="AfterBuild" DependsOnTargets="AfterBuild14;AfterBuild15;AfterBuildIns15"/>
```

Comment:
I'd suggest naming targets respectively: `PublishVS14`, `PublishVS15`, and `PublishVS15Insertable`